app/models/spree/payment_method/sisow_billing.rb

```diff
@@ -85,6 +92,8 @@ def payment_provider(transaction_type, options)
           return Sisow::BancontactPayment.new(options)
         when 'sofort'
           return Sisow::SofortPayment.new(options)
+        when 'paypalec'
```

Comment:
Indent `when` as deep as `case`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.